src/components/common/DocumentListItem.js

```diff
@@ -33,17 +33,27 @@ type Props = {
    * @param Id of item that was pressed.
    */
   onPress: (id: number) => void,
+  /**
+   * @desc Value of the Document List Item.
+   */
+  value?: string,
```

Comment:
Value naming is not clear, please change it to something more descriptive.

Reply:
Done